tests/unit/bokeh/test_tile_providers.py

```diff
@@ -12,9 +12,11 @@
 
 import pytest ; pytest
 
+# External imports
```

Comment:
slight import section reformatting weirdness. The `# External imports` comment above can be deleted, and the commented now-unused `#from bokeh._testing.util.api import verify_all` below should also be deleted

Reply:
I've removed both lines but I am not sure if `isort` won't complain. It does locally.